Sources/DotLottie/Public/Utils/Utils.swift

```diff
@@ -52,53 +54,116 @@ func fetchDotLottieAndWriteToDisk(url: URL) async throws -> URL {
     // Fetch data
     do {
         let data = try await fetchFileFromURL(url: url)
+
+        if let fileNameSubstring = url.lastPathComponent.split(separator: ".lottie").first {
+            let fileName = String(fileNameSubstring)
+            let urlBook = try writeDotLottieToDisk(dotLottie: data, fileName: fileName)
```

Comment:
as we are not using the urlBook, can just change to:

return try writeDotLottieToDisk(dotLottie: data, fileName: fileName)